main/ejml-core/test/org/ejml/data/TestDMatrixRMaj.java

```diff
@@ -204,4 +204,15 @@ protected DMatrixD1 createMatrix( int numRows, int numCols ) {
             assertTrue(Math.abs(mat.data[i] - orig.data[i]) > UtilEjml.TEST_F64);
         }
     }
+
+    @Test void get2DData() {
+        double[][] expected = {{0, 1}, {2, 3}};
+        double[][] actual = new DMatrixRMaj(expected).get2DData();
+
+        assertArrayEquals(expected, actual);
```

Comment:
Do you use IntelliJ? If so rules for code formatting are checked into the repo and you can format everything to meet the standards that way. Still working on automating that...

Reply:
Yup, option + cmd + L